renderer-macos/lib/stubs/BriskView.c

```diff
@@ -24,27 +24,31 @@ void ml_BriskView_insertSubview(NSView *view, NSView *child, intnat position) {
 
 void ml_BriskView_removeSubview(NSView *child) { [child removeFromSuperview]; }
 
-void ml_BriskView_setFrame(NSView *view, double x, double y, double w,
-                           double h) {
+void ml_BriskView_setFrame(
+  NSView *view, 
+  double x, 
+  double y, 
+  double w,
+  double h, 
+  double paddingLeft, 
+  double paddingRight, 
+  double paddingBottom, 
+  double paddingTop
```

Comment:
I don't like that we changed the signature for all elements even though it's only needed for `Text`. Couldn't it be set separately when we call `configureInstance`?

Reply:
No, it cannot. All layout properties have to be passed during the layout application pass.